src/components/BooksForm.js

```diff
@@ -8,7 +8,7 @@ const BooksForm = () => {
   const [title, setTitle] = useState('');
   const [category, setCategory] = useState('');
   const dispatch = useDispatch();
-  const handleForm = e => {
+  const handleSubmit = e => {
```

Comment:
For handleSubmit function, the title field should be cleared and category set to default value, after form submission. So, include this in the function:
 
```
this.setState({
    title: '',
    category: 'Action'
  });
```